### ISSUE TITLE
merge a few things from Rsenal

### ISSUE THREAD

**Issue by tim-salabim** [fdetsch/Orcs#2]
Moin, 
[Rsenal](https://github.com/environmentalinformatics-marburg/Rsenal) has become quite a beast (a fresh install takes about 30 min if not more) and most of the stuff is too specific for general use. I.e. the only three functions from Rsenal that I use regularly are 

* [`bumpVersion`](https://github.com/environmentalinformatics-marburg/Rsenal/blob/master/R/bumpVersion.R), 
* [`latticeCombineGrid`](https://github.com/environmentalinformatics-marburg/Rsenal/blob/master/R/latticeCombineGrid.R) and 
* [`latticeCombineLayer`](https://github.com/environmentalinformatics-marburg/Rsenal/blob/master/R/latticeCombineLayer.R). 

I am sure there are a few other general functions hidden in Rsenal that I can't recall right now. 

Orcs seems like a nice little package for general utility functions (mostly). Would you consider moving the above mentioned functions to Orcs? 
Maybe we could also consider a CRAN release of Orcs given that the functions are mostly for general use.

**Comment by fdetsch** [comment]
The package's title alone justifies a CRAN release! (:

But joking aside, this sounds like a good idea to me and **Orcs** has definitely fallen somewhat into oblivion lately. I will think about it (and also talk to the other guys during our upcoming **[satellite](https://cran.r-project.org/package=satellite)** meeting - in the end, **Rsenal** is a joint venture) and come back to you by the beginning of next week.

**Comment by tim-salabim** [comment]
Please also
* [`regressionStats`](https://github.com/environmentalinformatics-marburg/Rsenal/blob/master/R/regressionStats.R) - here we would need to modify the function a little to use a different lattice theme.
* [`se`](https://github.com/environmentalinformatics-marburg/Rsenal/blob/master/R/se.R)

**Comment by fdetsch** [comment]
* `bumpVersion`, 
* `latticeCombineGrid` and
* `latticeCombineLayer` 

are now part of the package (but also remain in **Rsenal** for reasons of backward compatibility). As regards the other two functions: although **Orcs** is considered omnidirectional, I would rather see `regressionStats` as part of a more regression-focused package like **[remote](https://github.com/environmentalinformatics-marburg/remote)**. What do you think about that? As regards `se`, similar functions are available e.g. from **FSA** ([`se`](https://www.rforge.net/doc/packages/FSA/se.html)) and **plotrix** ([`std.error`](https://rdrr.io/cran/plotrix/man/std.error.html)).

How shall we handle `:::` declarations in [`latticeCombineGrid`](https://github.com/fdetsch/Orcs/blob/master/R/latticeCombineGrid.R)? The `R CMD check` NOTE resulting therefrom will likely hamper an upcoming CRAN release.

**Comment by tim-salabim** [comment]
Yeah, `regressionStats` in **remote** could make sense... Re `:::`, I think these can simply be removed as they are exported S3 methods 
```r
> methods("c")
 [1] c.abIndex*        c.bibentry*       c.Date            c.difftime        c.GenericSummary* c.lazy_dots*      c.noquote         c.numeric_version
 [9] c.person*         c.POSIXct         c.POSIXlt         c.quosures*       c.quoted*         c.sfc*            c.sfg*            c.sparseVector*  
[17] c.trellis*        c.units*          c.warnings        c.yearmon*        c.yearqtr*        c.zoo*           
see '?methods' for accessing help and source code
> methods("update")
 [1] update,ANY-method         update,CHMfactor-method   update,mle-method         update,RasterBrick-method update,RasterLayer-method
 [6] update.corStruct*         update.default            update.FileProgressBar*   update.formula            update.gafs*             
[11] update.gls*               update.gnls*              update.groupedData*       update.lme*               update.lmList*           
[16] update.lmList4*           update.loglm*             update.merMod*            update.modelStruct*       update.nlme*             
[21] update.nlsList*           update.Package*           update.packageStatus      update.ProgressBar*       update.reStruct*         
[26] update.rfe*               update.safs*              update.TextStatusBar*     update.train*             update.trellis*          
[31] update.varComb*           update.varConstPower*     update.varExp*            update.varFunc*           update.varPower*         
see '?methods' for accessing help and source code
```
so should be available (though we may need a call to `requireNamespace`. I will investigate.

**Comment by fdetsch** [comment]
> Re `:::`, I think these can simply be removed as they are exported S3 methods

Not if the package is not loaded which could render an explicit call to `requireNamespace` necessary. From my current session: 

```{r}
> methods("c")
[1] c.bibentry*       c.Date            c.difftime        c.noquote         c.numeric_version c.person*         c.POSIXct         c.POSIXlt        
[9] c.warnings       
see '?methods' for accessing help and source code
```

**Comment by tim-salabim** [comment]
Can we additionally also migrate `compareDistributions` from Rsenal to Orcs?

**Comment by fdetsch** [comment]
Sure &ndash; the more utility functions are available from CRAN, the better. I flagged those functions that have already been migrated to **Orcs** as `deprecated()` in **Rsenal**, we should simply do the same for `compareDistributions()`. I'll take care of it within the next couple of days.

**Comment by fdetsch** [comment]
Available in 'develop' as of f57f02a.
